### ISSUE TITLE
permission denied  in custom command

### ISSUE THREAD

**Issue by shelllee** [ForkIssues/TrackerWin#1837]
I got the error `Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password)` when execute `git push origin xxxx:features/cloud` in custom command.

but it is ok in bash
```
xxxx MINGW64 /c/workspace/ (features/cloud)
$ git push origin xxxx:features/cloud
Enter passphrase for key '/c/Users/xxxx/.ssh/xxxxxxxx':
```
so is there a way to complete authority authentication automatically, or open a bash terminal and send command to it in custom command then I could fill in passward manually

**Comment by DanPristupov** [comment]
I just tried and I couldn't make it work myself. Looks this particular case is not handled properly in Fork (so, we can say it's a bug).

I'll try fix that in the next update.

**Comment by DanPristupov** [comment]
We just released Fork 1.83. Now custom commands and the internal commands must use the selected key from the 'Configure SSH keys' dialog.

**Comment by shelllee** [comment]
Thank you very much! My problem resolved with this release!

**Comment by shelllee** [comment]
> We just released Fork 1.83. Now custom commands and the internal commands must use the selected key from the 'Configure SSH keys' dialog.


Thank you very much! My problem was resolved with this release!

This issue could be closed.
